Sources/StreamChatUI/MessageActionsPopup/MessageActionsTransitionController.swift

```diff
@@ -258,6 +258,7 @@ open class ChatMessageActionsTransitionController: NSObject, UIViewControllerTra
         messageView.setUpLayoutIfNeeded(options: messageLayoutOptions, attachmentViewInjectorType: messageAttachmentInjectorType)
         messageView.channel = originalView.channel
         messageView.content = message
+        messageView.delegate = originalView.delegate
```

Comment:
This is required to allow the VoiceRecording ItemView to receive AudioPlayback events